src/JabbR-Core/Startup.cs

```diff
@@ -105,8 +118,37 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             app.UseStaticFiles();
             app.UseSignalR();
 
+            //var context = app.ApplicationServices.GetService<JabbrContext>();
+            //AddTestData(context);
            
         }
+
+        private void AddTestData(JabbrContext context)
```

Comment:
I was following a blog post experimenting with instantiating the context and populating it so we could use it with DI properly, but the problem in `base(options)` when it's instantiated that way still exists. I'll delete this, and we can come back to it when we meet with David Fowler or the EF team.